### PR TITLE
Feature/baseline results

### PR DESCRIPTION
**Pull request by saniemi**

### CODE REVIEW

File: DataScience/Analytics/beta/check_performance.py [hunk 1]
```diff
@@ -181,7 +194,9 @@ def _check_performance(data, verbose=True):
         print('Correct Match not in the Set:', len(incorrect))
 
     results.append(len(correct_in_set))
+    results.append(new_uprns)
```
Comment (Mironor) [inline DataScience/Analytics/beta/check_performance.py]: To the same array you append first an int (`len`) and then another array, is that correct ?

Comment (saniemi) [inline DataScience/Analytics/beta/check_performance.py]: Both are ints; line 171 shows that new_uprns = len() which returns an int. 

In general, this is not a great way of doing this, but it makes the plotting later easier. One could probably find a cleaner way of doing what's needed here (i.e. to capture results).

Comment (Mironor) [inline DataScience/Analytics/beta/check_performance.py]: I would rename `new_uprns` to `new_uprns_count` in this case to avoid confusion, up to you.

Comment (saniemi) [inline DataScience/Analytics/beta/check_performance.py]: Good point. Will change. Will also change not_matched to not_matched_count to be consistent.

**Pull request merged**